avocado-setup.py

```diff
@@ -267,6 +268,9 @@ def install_optional_plugin(plugin):
         pass
 
 
+
+
+
```

Comment:
not needed

Reply:
> @PraveenPenguin we also need to update README about these, right ?

yes doing same, just exploring how i can add an empty directory